themes/mlh.zsh-theme

```diff
@@ -12,17 +12,38 @@
 # # # Feel free to customize! # # #
 # # # # # # # # # # # # # # # # # #
 
-# To easily discover colors and their codes, type `spectrum_ls` in the terminal
+# To customize symbols (e.g MLH_AT_SYMBOL), simply set them as environment variables
+# for example in your ~/.zshrc file, like this:
+# export MLH_AT_SYMBOL=" at "
```

Comment:
No need to use `export`. Also you should document that the settings need to go before OMZ is sourced in the `.zshrc` file.

Reply:
thanks:)